### PR TITLE
useTableCore implemented to useSimpleList antd

### PR DESCRIPTION
**Pull request by ozkalai**
Hey, I'm client! Test me! 'MASTER'
[Link to ANTD-USE-SIMPLE-LIST-REFACTOR](https://antd-us-client.refine.pankod.com)

Test me! 'MASTER'
[Link to ANTD-USE-SIMPLE-LIST-REFACTOR](https://antd-us-refine.pankod.com)

Hey, I'm client! Test me! 'MASTER'
[Link to ANTD-USE-SIMPLE-LIST-REFACTOR](https://antd-us-client.refine.pankod.com)

Test me! 'MASTER'
[Link to ANTD-USE-SIMPLE-LIST-REFACTOR](https://antd-us-refine.pankod.com)

In this development, "useTableCore" is implemented to "useSimpleList"( antd ).

### CODE REVIEW

File: packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts [hunk 1]
```diff
@@ -19,13 +12,11 @@ import {
     HttpError,
     MetaDataQuery,
     LiveModeProps,
-    parseTableParams,
-    stringifyTableParams,
-    unionFilters,
-    setInitialFilters,
+    useTable as useTableCore,
 } from "@pankod/refine-core";
+import { useEffect } from "react";
```
Comment (omeraplak) [inline packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts:17]:
```suggestion
import { useEffect } from "react";
```
Can we move it to the top?

File: packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts [hunk 2]
```diff
@@ -85,86 +75,41 @@ export const useSimpleList = <
...
+        pageSize,
+        setFilters,
+        setCurrent,
+        setPageSize,
+        tableQueryResult: queryResult,
+    } = useTableCore({
+        resource: resourceFromProp,
+        initialSorter,
+        initialFilter,
+        permanentFilter,
```
Comment (omeraplak) [inline packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts:100]: There was also `permanentSorter` property in `useTable`. Can we add it here too?

Comment (salihozdemir) [reply packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts:100]: There were also `initialCurrent` and `initialPageSize` properties in `useTable`. You could add it too 👍

File: packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts [hunk 3]
```diff
@@ -85,86 +75,41 @@ export const useSimpleList = <
...
-                sorter,
-                filters,
-            });
-
-            return push(`${pathname}?${stringifyParams}`);
+        if (listProps.pagination && listProps.pagination.current) {
+            setCurrent(listProps.pagination.current);
+        }
+        if (listProps.pagination && listProps.pagination.pageSize) {
+            setPageSize(listProps.pagination.pageSize);
```
Comment (salihozdemir) [inline packages/antd/src/hooks/list/useSimpleList/useSimpleList.ts]: I think, if you pass the `initialCurrent` and the `initialPageSize` to `useTableCore`. This code will be unnecessary.

**Review by omeraplak — APPROVED** [review]

**Review by salihozdemir — APPROVED** [review]

**Pull request merged**